core/rawdb/taiko_state.go

```diff
@@ -0,0 +1,141 @@
+package rawdb
+
+import (
+	"encoding/binary"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+var (
+	taikoTailID        = []byte(":t:t-")
+	taikoIndexPrefix   = []byte(":t:o-")
```

Comment:
```suggestion
	taikoIndexPrefix   = []byte(":t:i-")
```
Should it be "i" instead of "o"?

Reply:
conflict with taikoStateIDPrefix.